code/go/0chain.net/smartcontract/zcnsc/mint.go

```diff
@@ -86,11 +106,20 @@ func (zcn *ZCNSmartContract) Mint(trans *transaction.Transaction, inputData []by
 		return
 	}
 
-	// verify signatures of authorizers
-	err = payload.verifySignatures(ctx)
+	var numAuth item
+	err = ctx.GetTrieNode(storagesc.ALL_AUTHORIZERS_KEY, &numAuth)
 	if err != nil {
-		msg := fmt.Sprintf("failed to verify signatures with error: %v, %s", err, info)
-		err = common.NewError(code, msg)
+		err = common.NewError(code, fmt.Sprintf("failed to get number of authorizers: %s; %s", err.Error(), info))
+		return
+	}
+
+	// verify signatures of authorizers
+	count := payload.countValidSignatures(ctx)
+	if count < int(gn.PercentAuthorizers)*numAuth.Field {
```

Comment:
`` percent_authorizers: 0.7``
you should make ``math.RoundToEven(gn.PercentAuthorizers*numAuth.Field)``
int(gn.PercentAuthorizers) will cut the float.